pkg/sbproxy/reconcile/reconcile_brokers.go

```diff
@@ -145,6 +153,10 @@ func (r *resyncJob) deleteBrokerRegistration(ctx context.Context, broker *platfo
 	}
 }
 
+func (r *resyncJob) brokerProxyName(broker *platform.ServiceBroker) string {
+	return fmt.Sprintf("%s%s-%s", r.options.BrokerPrefix, broker.Name, broker.GUID)
```

Comment:
CF-Proxy tests need to be adapted after adding the GUID to the broker proxy name